internal/cp/subsystem.go

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cp
+
+import (
+	"context"
+	"github.com/hazelcast/hazelcast-go-client/internal/cluster"
+	"github.com/hazelcast/hazelcast-go-client/internal/invocation"
+	"github.com/hazelcast/hazelcast-go-client/internal/logger"
+	iserialization "github.com/hazelcast/hazelcast-go-client/internal/serialization"
+)
+
+// Subsystem represent the CP service.
```

Comment:
Could you convert this doc lines to use `/*` format to be consistent with the rest of the code base?

Reply:
Refactored all doc comments at https://github.com/hazelcast/hazelcast-go-client/pull/901/commits/628c21c86995619aae201315bf28b9aef639102a.